src/util.js

```diff
@@ -146,3 +153,9 @@ exports.getLineAndColumnMessage = function(str, offset /* ...ranges */) {
   }
   return sb.contents();
 };
+
+exports.getLineAndColumnMessage = function(str, offset /* ...ranges */) {
```

Comment:
I think I'd get rid of the optional `ranges` here. This function can be the simple one that includes the line number and only points to a single offset, and any code that needs more than that can call the other version (which I've suggested you rename to `describeSourceLocation`.